mongo/options/clientoptions_test.go

```diff
@@ -240,6 +240,60 @@ func TestClientOptions(t *testing.T) {
 			})
 		}
 	})
+	t.Run("heartbeatFrequencyMS validation", func(t *testing.T) {
+		testCases := []struct {
+			name string
+			opts *ClientOptionsBuilder
+			err  error
+		}{
+			{
+				"heartbeatFrequencyMS > minimum (500ms)",
+				Client().SetHeartbeatInterval(10000 * time.Millisecond),
+				nil,
```

Comment:
Optional: Use keyed struct literals to make the test cases more readable.

E.g.
```go
{
	name: "heartbeatFrequencyMS > minimum (500ms)",
	opts:  Client().SetHeartbeatInterval(10000 * time.Millisecond),
	err:   nil,
}
```